hikari/channels.py

```diff
@@ -346,7 +384,7 @@ class TextChannel(PartialChannel, abc.ABC):
     __slots__: typing.Sequence[str] = ()
 
     # TODO: add examples to this and the REST method this invokes.
-    def history(
+    def fetch_history(
```

Comment:
Why rename this? `channel.history()` looks better :)

Reply:
If we ever add a method which gets the messages cached for this channel the name "fetch_history" gives us a distinction would be nice to have under the naming scheme present in #582

Reply:
Hmmm, true. Will mark this as a breaking change then